data/human/human missions.txt

```diff
@@ -6849,3 +6849,46 @@ mission "FW Refugees to Humanika"
 		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`
 
+mission "Returning to Paradise"
+	name "Take a family home to <planet>."
+	description "A family worried about the potential destruction within the Free Worlds wants to cut their vacation short and go home to <destination>."
+	source
+		government "Free Worlds"
+		attributes "tourism"
+	destination "Mainsail"
+	passengers 4
+	cargo "personal belongings" 1
+	to offer
+		random < 30
+		not "Rescue Katya 1: accepted"
```

Comment:
This condition doesn't exist. Checking for offered would be synonymous with checking if the mission were accepted (since you can't only either accept or defer the mission, and defer doesn't create any conditions).
```suggestion
		not "Rescue Katya 1: offered"
```

Although more generically, we could check if you've chosen sides.
```suggestion
		not "chosen sides"
```